gather-scheduler.js

```diff
@@ -112,6 +114,24 @@ class GatherScheduler {
       })
       .then(_ => artifacts);
   }
+
+  static saveAssets(saveTrace, tracingData, url) {
+    /* globals window */
+    function hostOfURL(url) {
+      if (typeof process !== 'undefined' && 'version' in process) {
+        return require('url').parse(url).hostname;
+      }
+      return new window.URL(window.location.href).hostname;
```

Comment:
actually, at least right now you shouldn't need this branch at all, since the extension doesn't have an option to set `saveTrace`